File: Python/1101 - Sequence of Numbers and Sum.py
while True:
    m, n = map(int, input().split())
    if m <= 0 or n <= 0:
        break
    soma = 0
    if m > n:
        m, n = n, m
    numeros = ""
    for i in range(m, n + 1):
        soma += i
        numeros += str(i) + " "
    print(f"{numeros}Sum={soma}")

File: Java/1101 - Sequence of Numbers and Sum.java
import java.util.Scanner;
public class Main
{
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		while(true) {
		    int m = sc.nextInt(),n = sc.nextInt(),soma = 0;
		    if (m<=0 || n<=0)
		        break;
		    if (n>m) {
		        int a = n;
		        n = m;
		        m = a;
		    }
		    for (int i = n; i <= m; i++) {
		        System.out.printf("%d ",i);
		        soma+=i;
		    }
		    System.out.printf("Sum=%d",soma);
		    System.out.println("");
		}
	}
}

